### ISSUE TITLE
Errors following line continuations reported on wrong line

### ISSUE THREAD

**Issue by nwhetsell** [csound/csound#553]
If I syntax-check this orchestra

```
prints \
"hello, world"error
```

by running in a Terminal

```sh
csound --syntax-check-only --orc Test.orc
```

then Csound reports the error in line&nbsp;1:

```
error: syntax error, unexpected T_IDENT, expecting NEWLINE or ','  (token "error") from file Test.orc (1)
 line 1:
>>>prints "hello, world"error <<<
```

I’d expect the error to be reported in line&nbsp;2.

For the sake of comparison, after attempting to run this similar Python script

```python
print\
("hello, world")error
```

Python 2.7.10 reports the error in line&nbsp;2:

```
  File "Test.py", line 2
    ("hello, world")error
                        ^
SyntaxError: invalid syntax
```

**Comment by jpffitch** [comment]
Very ad t fix unless the whole lexing code is rewritten, unless someone 
else can see a solution.

The issue is that there is no way to change the line number in 
mid-statement.

**Comment by jpffitch** [comment]
I have committed a patch; far from perfect but does get line right

**Comment by jpffitch** [comment]
Better patch removes artifacts of method

**Comment by nwhetsell** [comment]
Unfortunately, I’m not able to confirm this. With a fresh clone of the Csound repository after commit ce3f63e, building Csound on OS&nbsp;X stops at 75%. It looks like this has something to do with libwidgets.dylib:

```
[ 28%] Linking CXX shared library ../libwidgets.dylib
ld: library not found for -lpng
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [libwidgets.dylib] Error 1
make[1]: *** [InOut/CMakeFiles/widgets.dir/all] Error 2
```

Is there a new dependency for building Csound?

**Comment by jpffitch** [comment]
png is required for fltk as far as I know; try building without fltk?

**Comment by nwhetsell** [comment]
Oddly, I have libpng 1.6.18, the current version. CMake reports

```
-- Found PNG: /usr/local/lib/libpng.dylib (found version "1.6.18") 
```

I was able to compile Csound without difficulty after commit 9588534, which addressed issue #546. After another fresh clone of Csound (commit 8e16077), I’m now also seeing a link error with libsignalflowgraph.dylib:

```
[ 28%] Linking CXX shared library ../libsignalflowgraph.dylib
ld: library not found for -lpng
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [libvirtual.dylib] Error 1
make[1]: *** [InOut/CMakeFiles/virtual.dir/all] Error 2
```

Is it possible that something changed between commit 9588534 and commit ce3f63e that would’ve affected building on OS&nbsp;X?

How does one build Csound without FLTK?

**Comment by nwhetsell** [comment]
Never mind; somehow /usr/local/include was deleted from `cpp`’s search path. I can confirm that Csound reports the error in line&nbsp;2 on OS&nbsp;X.
